docs/configuration_file.md

```diff
@@ -143,6 +143,8 @@ Some options are also mandatory.
 
 * `low_priority_node_selector` (*optional*, `string`) — a nodeselector to be applied to builds, which are considered to have less priority than normal ones
 
+* `equal_labels` (*optional*, `string`) — list of equal labels groups, used for adding all equal labels into dockerfile if any is present, label delimiter ':', group delimiter ',' (e.g. `name1:name2:name3, release1:release2, version1:version2`)
```

Comment:
do you want to rename 'equal_labels' to the 'equal-preference_labels' or?

do you want me to just add your explanation or replace whole mine?